pkg/network/network.go

```diff
@@ -57,6 +58,8 @@ const (
 
 	// IstioOutboundIPRangesKey is the name of the configuration entry
 	// that specifies Istio outbound ip ranges.
+	//
+	// OBSOLETE: This will be completely removed in the future release.
```

Comment:
nit: Recommend using `DEPRECATED` instead of obsolete here as well. 

eg: https://github.com/knative/serving/blob/0767e1b5fac6d37e5b316d64f220e542654d16de/pkg/apis/serving/v1alpha1/revision_types.go#L96